financial-connections/src/main/java/com/stripe/android/financialconnections/features/bankauthrepair/BankAuthRepairScreen.kt

```diff
@@ -22,6 +22,8 @@ internal fun BankAuthRepairScreen() {
         onContinueClick = { /*TODO*/ },
         onCancelClick = { /*TODO*/ },
         onClickableTextClick = { /*TODO*/ },
-        onWebAuthFlowFinished = { /*TODO*/ }
-    ) { /*TODO*/ }
+        onWebAuthFlowFinished = { /*TODO*/ },
+        onViewEffectLaunched = { /*TODO*/ },
+        inModal = false
```

Comment:
This feature is not yet available in the SDK, but once it is, it'll display the partner auth pane full-screen.